ui/src/app/utils/subscribable.tsx

```diff
@@ -0,0 +1,43 @@
+import * as fp from 'lodash/fp';
+
+export interface Subscribable<T> {
```

Comment:
Could you add some basic docs to these interfaces? Method-level docs are probably not needed given the simplicity